CesiumGltf/include/CesiumGltf/AttributeSemantics.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <array>
+#include <string>
+
+namespace CesiumGltf {
+
+/**
+ * @brief The standard glTF attribute semantics from the specification.
+ */
+struct AttributeSemantics {
+  /**
+   * @brief Unitless XYZ vertex positions.
```

Comment:
Just a drive by comment, I think the glTF spec says the [linear distances](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#coordinate-system-and-units) are in meters? 😅 

![image](https://github.com/user-attachments/assets/8c596d13-3540-4a0b-abdc-54dd2caf9ce2)

Reply:
I copied "Unitless XYZ vertex positions" right out of the glTF specification. See the table in this section:
https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#meshes-overview

I think the documentation is not actually contradicting itself. The POSITION semantic is unitless, but after it's transformed by the Node hierarchy it's supposed to be meters.